ui/src/cloud/actions/limits.ts

```diff
@@ -34,3 +106,48 @@ export const getReadWriteLimits = () => async (
     }
   } catch (e) {}
 }
+
+export const getAssetLimits = () => async (
+  dispatch,
+  getState: () => AppState
+) => {
+  dispatch(setLimitsStatus(RemoteDataState.Loading))
+  try {
+    const {
+      orgs: {org},
+    } = getState()
+
+    const limits = (await getLimitsAJAX(org.id)) as Limits
+    dispatch(setLimits(limits))
+    dispatch(setLimitsStatus(RemoteDataState.Done))
```

Comment:
Might be nice to combine this and the above `setLimits` action into a single action, to avoid double renders due to the synchronous nature of Redux.